CRM/Membershipextrasimporterapi/EntityImporter/LineItem.php

```diff
@@ -313,4 +315,9 @@ private function getContributionFinancialTrxnId() {
     return $dao->financial_trxn_id;
   }
 
+  private function updateRelatedContributionAmount($lineItemTotalAmount) {
```

Comment:
Should we add some comment to this method as we have not just updated the total amount, we have a calculation in the sql statement as well?